source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -804,167 +800,27 @@ export default class WalletSendForm extends Component<Props, State> {
-                          }}
-                          allowSigns={false}
-                        />
-                        <div className={styles.rightContent}>
-                          {this.hasAssetValue(assetField) && (
-                            <div className={styles.clearAssetContainer}>
-                              <PopOver
-                                content={intl.formatMessage(
-                                  messages.clearLabel
-                                )}
-                                placement="top"
-                              >
-                                <button
-                                  onClick={() =>
-                                    this.clearAssetFieldValue(assetField)
-                                  }
-                                  className={styles.clearAssetButton}
-                                >
-                                  <SVGInline
-                                    svg={closeIcon}
-                                    className={styles.clearReceiverIcon}
-                                  />
-                                </button>
-                              </PopOver>
-                              <div className={styles.separator} />
-                            </div>
-                          )}
-                          <div className={styles.assetsDropdownWrapper}>
-                            <WalletsDropdown
-                              className={styles.assetsDropdown}
-                              {...assetsDropdownField.bind()}
-                              assets={sortedAssets}
-                              onChange={(newFingerprint) => {
-                                if (newFingerprint !== fingerprint) {
-                                  this.onChangeAsset(
-                                    fingerprint,
-                                    newFingerprint
-                                  );
-                                }
-                              }}
-                              syncingLabel={intl.formatMessage(
-                                messages.syncingWallet
-                              )}
-                              hasAssetsEnabled
-                              value={fingerprint}
-                              getStakePoolById={() => {}}
-                              errorPosition="bottom"
-                            />
-                          </div>
-                        </div>
-                      </div>
-                    );
-                  }
+                  (fingerprint: string, index: number) => (
+                    <AssetTokenInput
+                      key={fingerprint}
+                      fingerprint={fingerprint}
+                      index={index}
+                      getAssetByFingerprint={this.getAssetByFingerprint}
+                      availableAssets={this.availableAssets}
+                      receiverFormField={formFields.receiver}
```

Comment:
@yakovkaravelov we should not pass the whole `receiverFormField`. It should be enough to pass just the individual `assetField` and the `assetsDropdown`.